WordPress/Classes/ViewRelated/Blog/BlogListDataSource.swift

```diff
@@ -193,6 +226,8 @@ private extension BlogListDataSource {
         case editing
         case searching(String)
         case loggedIn
+        case matchingBlog(Blog)
+        case accountOwned
```

Comment:
I like this idea.  I honestly wasn't too sure what the loggedIn datasource was being used for (I assumed it was just the default blog list for a logged in user but didn't confirm). I'll make a few tweaks. :)